### DISCUSSION TITLE
[新功能需求] 增加部分组件及现有组件优化

### DISCUSSION THREAD

**Discussion by LTTGGj** [Moonofweisheng/wot-design-uni#58] [Ideas]
### 这个功能解决了什么问题？

分段器
返回顶部
横向滚动列表  
页面加载动画  
按钮自定义颜色及形状 
select下拉选择

### 你期望的 API 是什么样子的？

参考uview及element ui已有功能

**Comment by Moonofweisheng** [comment]
关于增加组件和组件优化内容，可见[讨论区](https://github.com/Moonofweisheng/wot-design-uni/discussions/45)
